src/AutoRest.CSharp/Mgmt/Models/MgmtRestOperation.cs

```diff
@@ -182,17 +184,17 @@ public MgmtRestOperation(MgmtRestOperation other, string nameOverride, CSharpTyp
 
         private CSharpType? GetFinalResponse()
         {
-            var finalSchema = Operation.LongRunningFinalResponse.ResponseSchema;
+            var finalSchema = Operation.LongRunning?.FinalResponse.BodyType;
```

Comment:
This is causing [additional internal rest operation](https://github.com/Azure/azure-sdk-for-net/pull/44148/files#diff-6dc9899281ca92ce54ccab5d43c123a817258d507983dd49e397475f839073a7).
With the[ swagger definition](https://github.com/Azure/azure-rest-api-specs/blob/9061674ac55af0ec23e12e4f9f505010ab428935/specification/automation/resource-manager/Microsoft.Automation/stable/2023-11-01/runbook.json#L146-L148), `ResponseSchema` from M4 is null, not same as `BodyType`.